<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TownView</class>
 <widget class="QWidget" name="TownView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 15, 15);
font: 14pt &quot;Ancient Modern Tales&quot;;
color: rgb(230, 230, 230);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>100</number>
     </property>
     <item>
      <widget class="ArmyLayout" name="wgt_heroArmy" native="true"/>
     </item>
     <item>
      <widget class="ArmyLayout" name="wgt_townArmy" native="true"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_stayAsGarrison">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Stay as garrison</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="ResourcesView" name="wgt_heroResources" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pb_exitTown">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ArmyLayout</class>
   <extends>QWidget</extends>
   <header>ui/army_layout.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResourcesView</class>
   <extends>QWidget</extends>
   <header>ui/resources_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
